RecoPixelVertexing/PixelTriplets/plugins/CAHitQuadrupletGenerator.cc

```diff
@@ -202,35 +239,215 @@ void CAHitQuadrupletGenerator::hitQuadruplets(const TrackingRegion& region,
         hitQuadruplets(region, result, hitDoubletsPtr, g, es);
         theLayerCache.clear();
 }
-
-void CAHitQuadrupletGenerator::hitNtuplets(const IntermediateHitDoublets::RegionLayerSets& regionLayerPairs,
-                                              OrderedHitSeeds& result,
+void CAHitQuadrupletGenerator::hitNtuplets(const IntermediateHitDoublets& regionDoublets,
+                                              std::vector<OrderedHitSeeds, std::allocator<OrderedHitSeeds> >& result,
```

Comment:
Remove explicit `std::allocator<OrderedHitSeeds>` as that is the default anyway.